test/sagemaker_tests/mxnet/training/__init__.py

```diff
@@ -11,3 +11,39 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
+
+import sagemaker
+from sagemaker.mxnet.estimator import MXNet as MXNet_origin
+
+
+class MXNet(MXNet_origin):
```

Comment:
Is there a reason we have implemented the MXNET retry in this way instead of the one added in TF? Also, Can we keep the mxnet estimator name as `MXNET` instead of `MXNet_origin`, we should rename the class name instead so that we can distinguish between a class and actual sagemaker estimator invocation

Reply:
MXNet tests are much simplier. Most of the time it's just _estimator.fit()_. I thought overwriting _fit()_ will safe a lot of efforts. 
Will rename the wrapper